.eslintrc

```diff
@@ -1,18 +0,0 @@
-env:
```

Comment:
ESLint now recommends using a file extension for your config file. See http://eslint.org/docs/user-guide/configuring#configuration-file-formats.

So this was renamed to .eslintrc.yml below.